syft/cpe/cpe.go

```diff
@@ -123,7 +125,7 @@ func normalizeField(field string) string {
 
 	// keep dashes and forward slashes unescaped
 	if field == "*" {
-		return wfn.Any
+		return Any
```

Comment:
Is this to decouple us from the `wfn` package in total?

Reply:
yeah, I didn't see a need to couple our implementation 👍